docs/analytics.md

```diff
@@ -0,0 +1,132 @@
+
+<i>Last updated: Feb 25, 2019</i>
+
+<br />
```

Comment:
Not sure you need the extra linebreaks in here, so I'd probably remove all these stray `<br />` tags. But they don't hurt anything, so :shrug: